files/jsondata/L10n-SVG.json

```
@@ -145,6 +160,7 @@
     "es": "Atributos de procesamiento condicional",
     "fr": "Attributs de traitement conditionnel",
     "ja": "条件処理属性",
+    "ko": "조건적 처리 속성",
```

```suggestion
    "ko": "조건부 처리 속성",
```

보통 이런 상황에서 부를 많이 사용해서 제안드립니다!